homeassistant/components/shopping_list/config_flow.py

```diff
@@ -0,0 +1,32 @@
+"""Config flow to configure ShoppingList component."""
+from homeassistant import config_entries
+
+from .const import DOMAIN  # pylint: disable=unused-import
+
+
+class ShoppingListFlowHandler(config_entries.ConfigFlow, domain=DOMAIN):
+    """Config flow for ShoppingList component."""
+
+    VERSION = 1
+    CONNECTION_CLASS = config_entries.CONN_CLASS_LOCAL_PUSH
+
+    def __init__(self):
+        """Init ShoppingListFlowHandler."""
+        self._errors = {}
+
+    async def async_step_user(self, user_input=None):
+        """Handle a flow initialized by the user."""
+        self._abort_if_unique_id_configured()
+
+        if user_input is not None:
+            return self.async_create_entry(title="Shopping List", data=user_input)
```

Comment:
I think you can remove `data=user_input`

Reply:
data is a required parameter for async_create_entry